README.md

```diff
@@ -36,6 +36,8 @@ etherlime ganache
 
 - `JSONRPCDeployer(privateKey, nodeUrl, defaultOverrides)` - given private key to the deployment wallet, the url to the node you are trying to connect (local or remote) and default deployment settings for `gasPrice` and `gasLimit` it exposes you a deployer object.
 
+- `EtherlimeGanacheDeployer(privateKey, nodeUrl, defaultOverrides)` - given private key to the deployment wallet, the url to the node you are trying to connect (local or remote) and default deployment settings for `gasPrice` and `gasLimit` it exposes you a deployer object.
```

Comment:
Write a more meaningful description of your deployer. 

Something like this deployer allows you to deploy on etherlime ganache with the first account. Then enumerate the optional params.

You can see here how optional params are written in brackets
https://github.com/yargs/yargs/blob/master/docs/api.md#completioncmd-description-fn